src/Java.Interop.Localization/Resources.resx

```diff
@@ -0,0 +1,277 @@
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="Generator_BG4000" xml:space="preserve">
+    <value>ERROR: failed to remove old constants: {0}.</value>
+    <comment>{0} - The list of constants that could not be removed.</comment>
+  </data>
+  <data name="Generator_BG4100" xml:space="preserve">
+    <value>Failed to process enum mapping. Text line: {0}.</value>
+    <comment>{0} - The line that could not be processed.</comment>
+  </data>
+  <data name="Generator_BG4200" xml:space="preserve">
+    <value>Error during processing metadata fixup: {0}.</value>
+    <comment>{0} - The error encountered.</comment>
+  </data>
+  <data name="Generator_BG4300" xml:space="preserve">
+    <value>Invalid XPath specification: {0}.</value>
+    <comment>{0} - The invalid XPath line.</comment>
+  </data>
+  <data name="Generator_BG4307" xml:space="preserve">
+    <value>Target attribute name is not specified for path: {0}.</value>
+    <comment>{0} - The invalid XPath line.</comment>
+  </data>
+  <data name="Generator_BG4400" xml:space="preserve">
+    <value>Unexpected 'global::' specification. This error happens if it is specified in the Metadata API fixup for example.</value>
+    <comment>Value in quotes should not be translated. "Metadata API" should not be translated.</comment>
```

Comment:
I'm happy with either approach, but just to minimize creativity and maximize familiarity for the translators for phrases to leave untranslated, this could instead use the boring template from xamarin-android:

```
The following are literal names and should not be translated: 'global::', Metadata API
```

On a different note, for message cleanup along the way, is `Metadata API` a literal name? That term doesn't appear under https://docs.microsoft.com/xamarin/android/. Would the following be accurate?

```
 <value>Unexpected 'global::' specification. This error happens if 'global::' is specified in the Metadata.xml transform file, for example.</value>
    <comment>The following are literal names and should not be translated: global::, Metadata.xml</comment>
```